### DISCUSSION TITLE
ERROR: The string is missing the terminator: '.

### DISCUSSION THREAD

**Discussion by maroschutte** [chocolatey/choco#2946] [General]
Are this ticket and its PR still on the radar (https://github.com/chocolatey/choco/issues/2410) ? It looks like a simple enough fix to adopt.

**Comment by m-schutte-ohra-nl** [comment]
Could one of the contributors please take a look at this?

**Reply by pauby** [reply]
The issue and PR have not been allocated to a milestone. Once that happens, we will pick it up for that release.

**Reply by m-schutte-ohra-nl** [reply]
Thanks for your reaction, I'll wait for that then
